manual/index-rst-template

```diff
@@ -3,9 +3,21 @@ Pytket User Manual
 
 This manual refers to the following versions of pytket and extensions:
 
+..
+    [comment]: if the versions are updated, update the version in the install-requirements and the requirements.txt, too.
+    [comment]: this duplication is currently necessary because of an issue with the pyjwt
+    [comment]: version required from qcs-api-client < 0.21.0
+
 .. code-block:: text
 
-REQUIREMENTS
```

Comment:
We probably don't need to create yet a third list that has to be updated. Can we just remove the seddery from `build-manual` and insert the list there, and delete `requirements.txt`? Would also need a change to `check_manual.yml`.

Reply:
(I guess `check_manual.yml` should just call `build-manual`?

Reply:
I think we have dont use `build-manual` in `check_manual.yml` because of the difference in the error handling and the destination of the build files.

I have now removed the requirements.txt. I think we might want to keep it like this because there might be requirements for notebooks that are not checked we want to include in this list?

If you see anything that I have missed, please let me know.